tests/views/resource_tests.py

```diff
@@ -351,7 +351,7 @@ def test_get_related_resource(self):
         ]
         reference_tile.save()
         sync_es(se)
-        ret = reference_resource.get_related_resources(user=user)
+        ret = reference_resource.get_related_resources(user=user, lang="en")
```

Comment:
@aarongundel looks like elastic wasn't this picky re: "en" vs. "en-US". Before I adjust the postgres query, can you help me look into why this was passing before when the value was "en" but the lookup (default arg to this function) was "en-US"?

Reply:
Never mind, I got it -- get_preflabel_from_conceptid() ranks labels, so I gotta keep parity with that.